SharedUtils/src/main/java/com/braintreepayments/api/TLSSocketFactory.java

```diff
@@ -118,7 +118,7 @@ private Socket enableTLSOnSocket(Socket socket) {
         if (socket instanceof SSLSocket) {
             ArrayList<String> supportedProtocols =
                     new ArrayList<>(Arrays.asList(((SSLSocket) socket).getSupportedProtocols()));
-            supportedProtocols.retainAll(Collections.singletonList("TLSv1.2"));
+            supportedProtocols.retainAll(Collections.singletonList("TLSv1.3"));
```

Comment:
We might still need TLS v1.2 as a fallback. It looks like v1.3 is [available in API 29+](https://developer.android.com/reference/javax/net/ssl/SSLSocket#protocols) (Android 10 and above).

Reply:
Great catch, I updated to include both 1.2 & 1.3 in 8416f42967ba92ab11131f3c9f649e103f52a9a2